.github/workflows/run-tests.yml

```diff
@@ -14,8 +14,8 @@ jobs:
         php: ['7.3', '7.4', '8.0']
         os: [ubuntu-latest, macos-latest]
         dependency-version: [prefer-lowest, prefer-stable]
+        phpunit: ['9.*']
         include:
-          - phpunit: '9.*'
           - php: '7.3'
             phpunit: '8.*'
```

Comment:
I know this isn't a change you made in this PR, but is it not weird that we have PHPUnit 8.* on the PHP, and Phpunit 9.* on its own line?

Reply:
`include` is supposed to be specifically for overrides and 'extra' stuff afaik, but PHPUnit is used in every test run and we just need to override the version that one time, so it should have had its own key in the `matrix` all along.

Reply:
👍 Thanks!

Reply:
@mattstauffer turns out I was very wrong about this! `include` is for 'extra' stuff but NOT for overrides. The previous syntax was correct so I've reverted it.